docs/release.md

```diff
@@ -0,0 +1,98 @@
+       --project gcp-variant-transforms-test
+   ```
+1. Wait until the test is done. It will take about 2 hours. Ensure that there
+   are no failures. In case of failures, rollback the version change in
+   `setup.py` and continue from Step #1 once the errors are fixed.
+1. To update the docker image, you can either:
+   * (**Preferred**) Copy the verified image to the main project by running the
+     following on your workstation. Note that you require
+     [docker](https://www.docker.com/) to be installed.
+
+     ```bash
+     gcloud auth configure-docker
+     SOURCE_IMAGE=gcr.io/gcp-variant-transforms-test/gcp-variant-transforms:"${COMMIT_SHA}"
+     TARGET_IMAGE=gcr.io/gcp-variant-transforms/gcp-variant-transforms:"${COMMIT_SHA}"
+     docker pull "${SOURCE_IMAGE}"
+     docker tag "${SOURCE_IMAGE}" "${TARGET_IMAGE}"
+     docker push "${TARGET_IMAGE}"
+     ```
+
+   * Push a new image from source by running:
+
+     ```bash
+     VERSION_TAG=v.X.X.X  # Replace with correct version tag!
+     RELEASE_FOLDER=$(mktemp -d)
+     mkdir -p "${RELEASE_FOLDER}"
+     cd "${RELEASE_FOLDER}"
+     git clone https://github.com/googlegenomics/gcp-variant-transforms.git
+     cd gcp-variant-transforms/
+     git checkout "${VERSION_TAG}"
+     gcloud container builds submit \
+         --config cloudbuild.yaml \
+         --project gcp-variant-transforms \
+         --substitutions _CUSTOM_TAG_NAME="${COMMIT_SHA}" .
+     ```
+1. Once the new image is published, navigate to the
+   [Container registry](https://console.cloud.google.com/gcr/images/gcp-variant-transforms/GLOBAL/gcp-variant-transforms?project=gcp-variant-transforms)
+   page and update the published image to have the labels `latest` and `X.X.X`
+   for the version number. You may optionally run a "sanity" test on the newly
+   released image by running:
+
+   ```bash
+   # TODO(arostami): modify deploy_and_run_tests.sh to use custom image.
+   temp_dir="$(mktemp -d)"
+   virtualenv "${temp_dir}"
+   source ${temp_dir}/bin/activate
+   pip install --upgrade .[int_test]
+   python gcp_variant_transforms/testing/integration/run_tests.py \
+       --project gcp-variant-transforms-test \
+       --staging_location "gs://integration_test_runs/staging" \
+       --temp_location "gs://integration_test_runs/temp" \
+       --logging_location "gs://integration_test_runs/temp/logs" \
+       --image gcr.io/gcp-variant-transforms/gcp-variant-transforms
+   ```
+1. Navigate to the
+   [releases tab](https://github.com/googlegenomics/gcp-variant-transforms/releases)
+   and click on **Draft a new release**.
+   1. Set the **Tag version** to `vX.X.X`
+      where `X.X.X` is the release version specified previously.
+   1. Set the **Target** to the commit that changed the version number.
+   1. Set the **Release title** to "Release vX.X.X".
+   1. Enter a description of the main features and/or bug fixes going into the
+      release.
+   1. Click on **Publish release**.
```

Comment:
Those asset files are generated automatically, nothing to do manually, right? .

Reply:
Correct.